CENNZnet_format.md

```diff
@@ -59,15 +59,52 @@ The domain is binary encoded. It contains a version definition, and a permission
                 * LE unsigned integer
         * If `constraints` is set
             * 1 byte
-                * Constraints length
-
+                * `constraints_length`
+                * LE unsigned integer
+                * `constraints_length` = byte_value + 1
+            * `constraints_length` bytes
+                * constraints payload
+* Contracts list
+    * 1 byte
+        * 1 bit
+            * `block_cooldown` presence
+        * 7 bits
```

Comment:
Would prefer to leave methods out for now as methods on contracts may require a different treatment

Reply:
Done